_posts/2015-05-28-API-Blueprint-GitHub.md

```diff
@@ -1,54 +1,53 @@
 ---
-title: API Blueprint on GitHub
+title: It's offical! API Blueprint and GitHub
 author: zdenek
-excerpt: "API Blueprint language as a first-class citizen on GitHub"
+excerpt: "API Blueprint recognized as a first-class citizen on GitHub"
 layout: post
-date: 2015-05-28 12:00:00
+date: 2015-06-03 12:00:00
 published: true
 comments: true
 ---
 
-Collaboration and continuous integration are the heart of every good API design. This is why we care about integration with CI services and GitHub. I am happy to announce the major milestone for API Blueprint.
+Collaboration and continuous integration are the heart of every good API design. This is why we care about integration with CI services and GitHub. 
 
-API Blueprint is now recognized as a language on GitHub!
+Today, I am proud to announce a major milestone for API Blueprint - it's now an official language on GitHub.
 
-All you need to do is to append the `.apib` extension to your blueprint file and push it to GitHub.
+To see the true beauty of API Blueprint on GitHib, you'll need to add `.apib` as the extension to your blueprint file and push to GitHub. 
```

Comment:
Oh I wouldn't say this .. (as the true beauty would be rendering it using aglio etc.)